stacks/dlaas/kubeflow/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Docker image for running examples in Tensorflow models.
+# base_image depends on whether we are running on GPUs or non-GPUs
+FROM hub.docker.com/clearlinux/stacks-dlaas-oss:latest
+
+RUN mkdir -p /opt
+RUN if [[ ! $(swupd bundle-list | grep go-basic) ]]; then swupd bundle-add go-basic --skip-diskspace-check; fi
```

Comment:
Uh I think go basic isn't in that image right? should just be able to do the bundle-add. Also not sure about why the diskspace checks are being skipped.